requirements/edx/base.txt

```diff
@@ -26,6 +26,7 @@ django-mptt>=0.8.6,<0.9
 django-oauth-toolkit==0.12.0
 django-pipeline-forgiving==1.0.0
 django-pyfs==1.0.7
+django-ratelimit==1.1.0
```

Comment:
@ormsbee don't we already have a way of doing this in the platform?

Reply:
Django DRF has its own thing, but I believe that only applies to DRF views. The solutions team came up with something back in the day, but I can't seem to find it any more -- maybe we got rid of it at some point?

Reply:
@mattdrayer: Did the solutions rate-limiting code get stripped out at some point?

Reply:
I don't believe we wrote custom rate-limiting code -- I think we leveraged the DRF rate limiting feature and handled it with configuration.  @saleem-latif may recall exactly how/where we implemented that.

Reply:
I wrote customized DRF Throttling for enterprise API (https://github.com/edx/edx-enterprise/blob/master/enterprise/api/throttles.py#L14), But it does not applies to non DRF views.